pkg/bgpv1/manager/reconcilerv2/service_test.go

```diff
@@ -1871,6 +1876,312 @@ func Test_ServiceVIPSharing(t *testing.T) {
 	}
 }
 
+func Test_ServiceAdvertisementWithPeerIPChange(t *testing.T) {
+	slog.SetLogLoggerLevel(slog.LevelDebug)
+	peerConfigs := []*v2.CiliumBGPPeerConfig{redPeerConfig}
```

Comment:
Please change `v2` to `v2alpha1` version for all `CiliumBGP*` resources in this test, that should resolve the build issue.

Reply:
Done. I've also fixed the slog reference keeping the podCIDRTestLogger as in the other tests